src/VisualStudio/Core/Def/ValueTracking/ValueTrackingToolWindow.cs

```diff
@@ -19,17 +22,29 @@ internal class ValueTrackingToolWindow : ToolWindowPane
     {
         private readonly ValueTrackingRoot _root = new();
 
-        public static ValueTrackingToolWindow? Instance { get; set; }
+        public bool Initialized { get; private set; }
+
+        [MemberNotNullWhen(returnValue: true, nameof(Initialized))]
```

Comment:
I believe you actually have to put this on the Initialized property and then you list all the other properties. It then means you can list all the properties at once.

Reply:
hmm... well that is different than what I thought but I'll give it a go!